BrCredentialCreatorWizard.vue

```diff
@@ -81,32 +91,24 @@ export default {
       type: Array,
       default: () => [],
       required: true
+    },
+    welcome: {
+      type: Object,
+      default: () => undefined
+    },
+    setup: {
```

Comment:
What is this? Is this always set up? Or is this "pre-flow"? If the latter why can't this be included in the flow array that we have already?

Reply:
It is "pre-flow" (and I considered calling it that, but didn't like the hypen :)). It can't be included in the `flow` unless we figure out another way to enable customization of elements for those particular steps. We could specify a `type` in the steps that allow for a custom representation of that step (another option I considered). If the type was set to X then we could use the slot instead of the default elements/component.

Note: I didn't want to use a property in the steps like "component" because it binds our "flows" too strongly to a particular implementation.